libs/banklessdao/util-data/src/http/http.ts

```diff
@@ -45,6 +45,7 @@ const decodeResponse = <T>(codec: t.Type<T>) => {
             pipe(
                 codec.decode(data),
                 E.mapLeft((e: t.Errors) => {
+                    console.log(`=== data: ${data} (${typeof data}) ===`);
```

Comment:
Is this left over from debugging or on purpose?

Reply:
Yes, fixed.